### DISCUSSION TITLE
xarray.Dataset.to_zarr:  docs for "Appending to existing Zarr stores”???

### DISCUSSION THREAD

**Discussion by nasa-nccs-cds** [pydata/xarray#5195] [Q&A]
In the description of the '`region`' argument to `xarray.Dataset.to_zarr` the last sentence states: _See “Appending to existing Zarr stores” in the reference documentation for full details_. I have not been able to find this reference in the reference docs or anywhere else- I'm starting to suspect that it doesn't exist. Can anyone point me to a location for this alleged documentation?

**Comment by keewis** [comment]
here it is: https://xarray.pydata.org/en/stable/io.html#appending-to-existing-zarr-stores. Since we started to link to the narrative documentation in the *See Also* section, I guess we could add a link there, too.
